### PR TITLE
Ad company name, dependencies and assets

### PR DESCRIPTION
**Pull request by zucher**
This code allows to add assets company name and dependencies to a nativefier project in roder to enhance an existing web site with local scripts and ressources.

### CODE REVIEW

**Comment by ronjouch** [comment]
@zucher do you still want to merge this? If yup, please:

- fix the lint errors (check for errors with `npm run lint`)
- rebase (note that https://github.com/jiahaog/nativefier/pull/417 worked on `companyName`, you might have conflicts)
- and re-push.

**Comment by zucher** [comment]
@ronjouch Is usage of Promise possible ?, in that case, can I move the project to be ES6 compliant ?

**Comment by zucher** [comment]
@ronjouch what about code duplication, I have the same code between buildApp and gulp helper

**Comment by ronjouch** [comment]
> Is usage of Promise possible ?, in that case, can I move the project to be ES6 compliant ?

@zucher the more ES6, the merrier, as long as our CI passes on Node 4.x

**Comment by zucher** [comment]
@ronjouch seems ok now ;-)

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "page-icon": "^0.3.0",
     "progress": "^2.0.0",
     "sanitize-filename": "^1.5.3",
-    "shelljs": "^0.7.0",
+    "shelljs": "^0.7.8",
```
Comment (ronjouch) [inline package.json]: @zucher alright to bump this dep if you need it, and thanks for adding `eslint-plugin-promise`. But if you modify `package.json` you need to regenerate, commit, and push [`package-lock.json`](https://docs.npmjs.com/files/package-lock.json) too. If you're not familiar with it, the TL;DR is: `rm -rf node_modules package-lock.json && npm install && git add package-lock.json && git commit -m "Regenerate package-lock.json"`

Comment (zucher) [reply package.json]: Where come from package-lock.json, with npm install I don't generate the package-lock.json ?

Comment (ronjouch) [reply package.json]: @zucher ah, you must be on an old version of `npm`. Don't worry about it, then, I'll do it. Keep me posted about the documentation, then I will get your branch, add a single commit for the package-lock thingie, and merge.

Comment (zucher) [reply package.json]: Thank you

**Comment by zucher** [comment]
Seems to be ok now @ronjouch, except for the npm version.

Regards

File: docs/api.md [hunk 2]
```diff
@@ -408,6 +410,69 @@ Sets a default zoom factor to be used when the app is opened, defaults to `1.0`.
...
+```
+--assets
+```
+
+Allow to specify additional assets array dedicated to the desktop version of your electron application, might be either a file or a directory
+
+Example:
+
+```bash
+nativefier http://google.com --assets ./assets/mainInjector.js --assets ./images ....
```
Comment (gootik) [inline docs/api.md]: shouldn't the command ordering have options first? i.e.  `nativefier --assets ... http://google.com`

**Comment by voltechs** [comment]
> Seems to be ok now @ronjouch, except for the npm version.
> 
> Regards

@zucher, is this ready for another review? Looks like @ronjouch has requested changes (I cannot predict if changes since satisfy those requests).

**Comment by SnowballSH** [comment]
Do you still plan to work on this feature? It would be great if we can modify the company name!

**Pull request closed without merging**